starport/cmd/scaffold_module.go

```diff
@@ -42,15 +42,19 @@ func NewScaffoldModule() *cobra.Command {
 }
 
 func scaffoldModuleHandler(cmd *cobra.Command, args []string) error {
-	s := clispinner.New().SetText("Scaffolding...")
-	defer s.Stop()
-
 	var (
 		options []scaffolder.ModuleCreationOption
 
 		name    = args[0]
 		appPath = flagGetPath(cmd)
 	)
+	sc, err := newApp(appPath)
+	if err != nil {
+		return err
+	}
+
+	s := clispinner.New().SetText("Scaffolding...")
```

Comment:
It's better if we print these spinners before any IO (which newApp does) otherwise, it'll be displayed with a delay.

Reply:
The spinner will be stuck at the beginning of the message. I see three ways to solve the issue
1 - Keep `newApp` before the spinner. The delay is imperceptible.
2 - Return a [final message variable](https://github.com/briandowns/spinner#final-string-output) from `newApp`, and set into the spinner.
3 - We can put the `newApp` after the spinner and forget about the `Scaffolding...` message stuck at the beginning of the warning.

![Screen Shot 2021-09-20 at 13 25 56](https://user-images.githubusercontent.com/2406457/134038553-
97285f80-1e8e-4f67-be12-3a3af8d6dc15.png)

Reply:
Oh, I see, that's why. What about if we pass an instance of the spinner to newApp and it can Stop() before printing and Start() after?

Otherwise, if the delay is negligible we can just keep it as it's. Whichever you feel is the best. 👍

Reply:
done on https://github.com/tendermint/starport/pull/1550/commits/0d473db51d2797d0df562b77f6bf98887e978d18